test/unit/test_grid_refinement.py

```diff
@@ -687,7 +687,7 @@ def test_mortar_grid_1d_refine_1d_grid(self):
 
             gb.update_nodes({old_g: new_g})
             mg = d["mortar_grid"]
-            mg.update_slave(new_g, 1e-4)
+            mg.update_secondary(new_g, 1e-4)
 
             high_to_mortar_known = np.matrix(
```

Comment:
secondary_...

Reply:
I'm in favor of renaming the methods`update_primary` and `update_secondary` by `update_primary_grid` and `update_secondary_grid`. What do you think?

Reply:
Sorry, my comment was unclear. I was thinking about secondary_to_mortar_known (or should perhaps be primary_to_mortar_known). But yes, I agree with your suggestion

Reply:
TODO: See #478